src/iam/chainConfig.ts

```diff
@@ -1,18 +1,24 @@
 import { VoltaAddress1056 } from "@ew-did-registry/did-ethr-resolver";
 import { CacheServerClientOptions } from "../cacheServerClient/cacheServerClient";
+import { MessagingMethod } from "../utils/constants";
```

Comment:
It's also a little confusing that the option is `CacheServer` but they need to provide a `natsServerUrl`. Should the option be `NATS` instead of `CacheServer`?

Reply:
Yeah I think the naming is not correct, cause as I said, cacheServer !== nats server